### ISSUE TITLE
Reading array query parameter in `[]time.Time`?

### ISSUE THREAD

**Issue by ganigeorgiev** [go-ozzo/ozzo-routing#61]
Reading array query parameters doesn't seem to work with common non primitive types (eg. `[]time.Time`, `[]uuid.UUID`, etc.).

Example:

```
// url: http://localhost:8090/api/test?Names=test1&Dates=2020-10-11T21%3A00%3A00.000Z

data := &struct{
    Names []string
    Dates []time.Time
}{}

ctx.Read(data) // result: { Names: [ "test1" ], Dates: [] }
```

_I guess if an alias type with custom `encoding.TextUnmarshaler` is provided it would work, but that seems redundant because the non slice version is correctly unmarshalized._

**Comment by kolkov** [comment]
Yeah, the query parameter reader uses Go's basic type conversion under the hood, so it doesn't handle types like `time.Time` or `uuid.UUID` that need custom parsing. Implementing `encoding.TextUnmarshaler` on a wrapper type is actually the right approach here — it's a bit verbose but it's the standard Go way.

Is this still relevant for you? Let us know, otherwise we'll close this in a couple of weeks.

**Comment by ganigeorgiev** [comment]
@kolkov You've replied with 30+ LLM generated comments in the span of 5min. I have  no idea why you were given maintainer access but if you believe that this is OK, then I really don't want to collaborate with you.

**Comment by kolkov** [comment]
@ganigeorgiev I hear you on the notification volume. These repos were abandoned for 6 years with 80+ stale issues across them — batch triage was the practical way to find out what's still relevant. Next time I'll spread it out.

My main focus is on [GoGPU](https://github.com/gogpu) and [Born ML](https://github.com/born-ml/born), and I've also been building modern successors to some ozzo libraries at [coregx](https://github.com/coregx). But go-ozzo itself is still important to me — a number of my production systems rely on these libraries. Qiang Xue trusted me with maintaining them, and I take that seriously.

Your original issue with `[]time.Time` query params is valid though. Feel free to reopen if it's still relevant.
